docs/articles/documentation/reference/api/runner/browsers.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: docs
+title: Runner.browsers Method
+permalink: /documentation/reference/api/runner/browsers.html
+---
+# Runner.browsers Method
+
+Configures the test runner to run tests in the specified browsers.
+
+```text
+browsers(browser) → this
+```
+
+The `browser` parameter can be any of the following objects or an `Array` of them:
+
+Parameter Type                                                                                        | Description                            | Browser Type
+---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------
+String &#124; Array                                                                                           | A different browser alias for each browser type. See [Browser Support](../../../concepts/browsers.md) for more details.                            | [Local browsers](../../../concepts/browsers.md#locally-installed-browsers), [cloud browsers](../../../concepts/browsers.md#browsers-in-cloud-testing-services), and [browsers accessed through *browser provider plugins*](../../../concepts/browsers.md#nonconventional-browsers).                                                                 |
+ `{path: String, cmd: String}`                                                                        | The path to the browser's executable (`path`) and command line parameters (`cmd`). The `cmd` property is optional.                                                     | [Local](../../../concepts/browsers.md#locally-installed-browsers) and [portable](../../../concepts/browsers.md#portable-browsers) browsers
+[BrowserConnection](../browserconnection/README.md)                                                            | The remote browser connection.                                                                                                                                        | [Remote browsers](../../../concepts/browsers.md#browsers-on-remote-devices)
+
+You do not need to call this function if you specify the [browsers](../../configuration-file.md#browsers) property in the [configuration file](../../configuration-file.md).
+
+*Related configuration file property*: [browsers](../../configuration-file.md#browsers)
+
+## Use Browser Aliases
+
+* running local browsers
+
+```js
+runner.browsers(['safari', 'chrome']);
+```
+
+* running browsers accessed through browser provider plugins
+
+```js
+runner.browsers('saucelabs:Chrome@52.0:Windows 8.1');
+```
+
+## Specify the Path to the Browser Executable
+
+Use the `path:` prefix. Enclose the path in backticks if it contains spaces.
```

Comment:
The first sentence does not sound good. I think repetition won't hurt here, i.e. "use the prefix to.. (achieve this)"